starfyre/component.py

```diff
@@ -17,9 +17,13 @@ class Component:
     html: str = ""
     css: str = ""
     js: str = ""
-    # on any property change, rebuild the tree
     original_name: str = ""
+    # on any property change, rebuild the tree
+    
     
+    def __post_init__(self):
```

Comment:
Let's not do this. We can pass the `original_name` during the initialization of the component. 

This will override a lot of things

Reply:
> Otherwise, we will have to change every place of code where we create an instance of Component? I found it in at least 8 places in our code, so it is more code duplication.

This is not duplication but us conforming to the dataclass schema. Using `post_init`, we are doing things implicitly, which then kills the purpose of having a dataclass.